core/src/test/java/org/hibernate/ogm/backendtck/associations/onetoone/OneToOneTest.java

```diff
@@ -163,7 +167,18 @@ public void testBidirectionalOneToOneCompositeId() throws Exception {
 
 	@Override
 	protected Class<?>[] getAnnotatedClasses() {
-		return new Class<?>[] {
+
+		if ( SkipByHelper.skipForGridDialect( GridDialectType.ORIENTDB ) ) {
```

Comment:
Why this if?
The purpose of this tests is to be run on all dialects or eventually excluded for some dialect if there is some incompatibility. Couldn't you create a separate test compatible with OrientDB and skip this one for OrientDB?